sympy/calculus/util.py

```diff
@@ -840,6 +840,130 @@ def minimum(f, symbol, domain=S.Reals):
     else:
         raise ValueError("%s is not a valid symbol." % symbol)
 
+def maxima(f, symbol, domain=S.Reals):
+    """
+    Returns the maxima of a function in the given domain.
+
```

Comment:
Perhaps it should be explained that these are local maxima, and not even the maximum function values but the argument values where the local maxima are attained.